### PR TITLE
Sequelize

### PR DESCRIPTION
**Pull request by nianokia**
# Please Review ClientForm.jsx & server.js

_You can choose to check the form, the logic, or both_

### _Relevant Info:_
* Used [Sequelize](https://sequelize.org/docs/v6/getting-started/) to connect and send requests to my database.
* POST request is sent to the database locally so it could result in an error for reviewers since there's no db dump file

## Check Form Submission
* Please open dev tools and verify that the inputted data from the client form prints to the console.
    * When I submit the form locally the console prints the data and sends it to the database on submit.

demo gif of form submission

## Check Logic / Give Suggestions to Improve Legibility / Revise code to Follow Best Practices
* Is the logic sound?
* Is it readable?
* Does it follow best practices?

### CODE REVIEW

**Comment by nianokia** [comment]
@margaretgeary Hi Margaret! Could you review the PR and tell me your thoughts?

File: client/src/components/ClientForm.jsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ const ClientForm = () => {
 
   const postClientEntry =  async (newClientEntry) => {
     // doesn't currently fetch anything since there's no POST request in the server yet, that's why there a fetch error in the console
```
Comment (margaret-ellevation) [inline client/src/components/ClientForm.jsx]: Can remove this comment!

File: server/server.js [hunk 2]
```diff
@@ -2,15 +2,114 @@ import express from 'express';
...
+    //     first_name: req.body.first_name,
+    //     last_name: req.body.last_name,
+    //     email: req.body.email,
+    //     type: req.body.type,
+    //     issue: req.body.issue,
+    //     age: req.body.age,
+    //     race: req.body.race,
+    //     gender: req.body.gender,
+    //     comment: req.body.comment
+    // }
```
Comment (margaret-ellevation) [inline server/server.js]: Can remove commented out lines

File: server/server.js [hunk 3]
```diff
@@ -2,15 +2,114 @@ import express from 'express';
...
+
+    try {
+        const newClient_Entry = await Client_Entry.create({
+            first_name, last_name, email, type, issue, age, race, gender, comment
+        });
+        console.log("Succesfully created entry!");
+        console.log(newClient_Entry);
```
Comment (margaret-ellevation) [inline server/server.js]: Can remove console.logs

**Review by margaret-ellevation — COMMENTED** [review]
Looks great! I only have a few comments on removing console.logs/commented lines. Great job otherwise!

**Comment by margaret-ellevation** [comment]
Feel free to re-request a review once you make edits and I can approve.

File: server/package-lock.json
Comment (abbiehuynh) [inline server/package-lock.json:1]: This is optional, but you do not have to commit this file, it can be added to your .gitignore.

**Comment by abbiehuynh** [comment]
Your code looks well-organized and was easy to read and follow the logic. Love that you were able to utilize Sequelize. I look forward to seeing what more you will accomplish!

File: client/src/components/ClientForm.jsx (hunk 1 above)
Comment (nianokia) [reply client/src/components/ClientForm.jsx]: Removed! Nice call out!

File: server/server.js (hunk 2 above)
Comment (nianokia) [reply server/server.js:107]: Removed!

File: server/server.js (hunk 3 above)
Comment (nianokia) [reply server/server.js]: Removed!

File: server/server.js (hunk 2 above)
Comment (nianokia) [reply server/server.js]: Removed!

File: server/package-lock.json
Comment (nianokia) [reply server/package-lock.json:1]: I didn't know this was optional. Thanks for the suggestion! It's now in my .gitignore.

**Comment by nianokia** [comment]
> Your code looks well-organized and was easy to read and follow the logic. Love that you were able to utilize Sequelize. I look forward to seeing what more you will accomplish!

Thank you so much again! You taught me something new today in regard to the package-lock.json, so I appreciate it!

**Comment by nianokia** [comment]
> Looks great! I only have a few comments on removing console.logs/commented lines. Great job otherwise!

Thanks for reviewing the code! It looks a lot cleaner now without the extra comments. 

> Feel free to re-request a review once you make edits and I can approve.

I added some additional comments to further clarify the code. With that, I think I was able to implement Sequelize sufficiently, so I'll go ahead and merge this in!

**Comment by nianokia** [comment]
## Summary of Pushed up Files
* ClientForm.jsx
* AdminListEntries.jsx
* server.js
* package.json
* package-lock.json
* .gitignore

**Pull request merged**